README.md

```diff
@@ -1 +1,27 @@
-# animal-stories
\ No newline at end of file
+# animal-stories
+
+# detect if a word is an animal
+
+animal_list = ["hare", "rabbit", "tortoise", "goat", "hound", "duck", "ducking", "fox", "lion", "boar", "ant", "grasshopper", 
+                "cat", "monkey", "dog", "mouse", "crow", "eagle", "wolf", "peacock"]
+
+def isAnimal(value):
+    if value in animal_list:
+        value = True    
+    else:
+        value = False
+        
+isAnimal("hare")
+isAnimal("carrot") 
+isAnimal("rabbit")
+
+
+# count words
+
+def countWords(text):
+    word_list = text.split()
+    dic = {}
+    for word in word_list:
+        dic[word] = dic.get(word, 0) + 1
+    
+countWords("i like carrots like rabbits")
```

Comment:
Same issue: print(countWords("....")) prints out None. You forgot to return a value.

Reply:
Also, we usually write one pull request per task. This helps work on several things at the same time. For example if countWords works but isAnimal does not yet, you can merge the first pull request while you keep working on the other.